src/Unosquare.RaspberryIO.Peripherals/Point3d.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Unosquare.RaspberryIO.Peripherals
+{
+    /// <summary>
+    /// 3D-space point.
+    /// </summary>
+    public struct Point3d
```

Comment:
This seems like a vector, review the [Vector3 Struct](https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector3?view=netframework-4.8)